src/components/grid_add_rows_footer/grid_add_rows_footer.xml

```diff
@@ -4,7 +4,12 @@
       class="o-grid-add-rows mt-2 ms-2 w-100 d-flex position-relative align-items-center"
       t-att-style="addRowsPosition"
       t-on-mousedown.stop.prevent="">
-      <button t-on-click="onConfirm" t-att-disabled="state.errorFlag" class="o-button">Add</button>
+      <button
+        t-on-click="onConfirm"
+        t-att-disabled="state.errorFlag"
+        class="o-button o-button-grey">
```

Comment:
If every single button with the class `o-button` should also have the class `o-button-grey`, what's the point of the class `o-button-grey` ? Just includes it in the class `o-button`. Just make it so `btn-outline-danger` correctly overwrites `o-button`

Reply:
I chose to go with the same approach Bootstrap seems to have , split the form from the colors which is better if we want to have a clean support of different color schemes in the future.
The runbot "red" has nothing to do with this PR. I started a rebuild and we'll see